src/components/ComponentCard.jsx

```diff
@@ -3,17 +3,18 @@ import Link from 'next/link';
 export default function ComponentCard({ name, slug, image }) {
   return (
     <Link href={slug}>
-      {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
-      <div className="component-card" tabIndex={0}>
-        <span className="component-card__header">
-          {name}
-        </span>
-        <img
-          src={image}
-          alt={`example of ${name} element`}
-          className="component-card__image"
-        />
-      </div>
+      <a className="component-link">
+        <div className="component-card">
+          <span className="component-card__header">
+            {name}
+          </span>
+          <img
+            src={image}
+            alt={`example of ${name} element`}
```

Comment:
I think these alt tags can be empty - the images in the component cards are decorative and I think the names of the components are sufficient here.